### ISSUE TITLE
Unable to load the custom pretrained model . How to test after pretraining the bert model ?

### ISSUE THREAD

**Issue by ajbot2019** [google-research/bert#953]
Hi  [Pre-training with BERT](https://github.com/google-research/bert#pre-training-with-bert)

i ran these 2 file as mentioned in the instruction ,
python create_pretraining_data.py
python run_pretraining.py 

after running those 2 file  i got below list of files in the folder output_dir=/tmp/pretraining_output

checkpoint				     
eval					     
eval_results.txt			 
events.out.tfevents.1575545953.26760d2fc979  
graph.pbtxt				    
model.ckpt-0.data-00000-of-00001
model.ckpt-0.index
model.ckpt-0.meta
model.ckpt-20.data-00000-of-00001
model.ckpt-20.index
model.ckpt-20.meta

1.what should i do now to test and perform the   prediction of  masked words for the sentences in the  sample_text.txt that was used as corpus to pretrain this masked LM .

i know how we can load the BertForMaskedLM to do the mask word prediction but how to do it for this pretrained model that i have got in the output_dir=/tmp/pretraining_output ?

when i tried to load the model from this output instead of using BertForMaskedLM.from_pretrained('bert-base-uncased')  i got below errror 

from transformers import BertForMaskedLM
BertNSP=BertForMaskedLM.from_pretrained('/content/drive/My Drive/bert_training/pretraining_output/')

when i import 
Model name '/content/drive/My Drive/bert_training/pretraining_output/' was not found in model name list (bert-base-uncased, bert-large-uncased, bert-base-cased, bert-large-cased, bert-base-multilingual-uncased, bert-base-multilingual-cased, bert-base-chinese, bert-base-german-cased, bert-large-uncased-whole-word-masking, bert-large-cased-whole-word-masking, bert-large-uncased-whole-word-masking-finetuned-squad, bert-large-cased-whole-word-masking-finetuned-squad, bert-base-cased-finetuned-mrpc, bert-base-german-dbmdz-cased, bert-base-german-dbmdz-uncased). 

We assumed '/content/drive/My Drive/bert_training/pretraining_output/config.json' was a path or url to a configuration file named config.json or a directory containing such a file but couldn't find any such file at this path or url.

**Comment by danyaljj** [comment]
Did you figure this out?

**Comment by karimomaya** [comment]
Did anyone figure this out?
